### PR TITLE
docs: 102 Configuration

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horacek <phoracek@redhat.com>

**What this PR does / why we need it**:

Documentation of the positive flow.

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: docs/user-guide-102-configuration.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+
+```
+NAME              STATUS
+bond0-eth1-eth2   SuccessfullyConfigured
+```
+
+By using `-o yaml` you obtain the full Policy with its current state:
+
+```shell
+kubectl get nncp bond0-eth1-eth2 -o yaml
```
Comment (qinqon) [inline docs/user-guide-102-configuration.md]: Maybe we can do -o=jsonpath={.status} so we don't have to say nothing about runcate

File: docs/user-guide-102-configuration.md [hunk 2]
```diff
@@ -0,0 +1,443 @@
...
+        dhcp: false
+        enabled: false
+      mac-address: 52:55:00:D1:56:01
+      mtu: 1500
+      name: eth2
+      state: up
+      type: ethernet
+```
+
+As you can see, the configuration is indeed applied and there is a bond
```
Comment (qinqon) [inline docs/user-guide-102-configuration.md]: I think we cannot refer to "you" wi ould remove "you" from all the doc.

File: docs/user-guide-102-configuration.md (hunk 1 above)
Comment (phoracek) [reply docs/user-guide-102-configuration.md]: That returns JSON, so it is impossible to read in this size

File: README.md [hunk 3]
```diff
@@ -37,9 +37,11 @@ You can choose to deploy this plugin on a
 
 Following comprehensive 101 series is the best place to start learning about all the features of this operator:
 
-1. [State](docs/user-guide-101-reporting-state.md) -
+1. [Reporting](docs/user-guide-101-reporting-state.md) -
    observe the current state of network on cluster nodes.
-2. Stay tuned for more!
+2. [Configuration](docs/user-guide-102-configuration.md) -
```
Comment (fedepaol) [inline README.md]: "Configuring", to keep consistent with the one above?

File: docs/user-guide-101-reporting.md [hunk 4]
```diff
@@ -193,4 +193,7 @@ kubectl -n nmstate delete pods --all
 
 The `NodeNetworkState` will now list all interfaces seen on the host.
 
-<!-- TODO: Link the next introduction article once it is introduced -->
+## Continue reading
+
+Following tutorial will guide you through configuration of node networking:
```
Comment (fedepaol) [inline docs/user-guide-101-reporting.md]: The following tutorial ... through THE configuration of ...

File: test/e2e/demos_test.go [hunk 5]
```diff
@@ -0,0 +1,34 @@
+// This module is meant to cover all the demos as we show them. To make it as close
```
Comment (fedepaol) [inline test/e2e/demos_test.go]: Why this file belongs to this commit?

Comment (phoracek) [reply test/e2e/demos_test.go]: It tests the tutorial, following the same steps and using same examples to verify that it indeed works and we don't make fools of ourselves in the docs.

File: docs/user-guide-101-reporting.md [hunk 6]
```diff
@@ -193,4 +193,7 @@ kubectl -n nmstate delete pods --all
 
 The `NodeNetworkState` will now list all interfaces seen on the host.
 
-<!-- TODO: Link the next introduction article once it is introduced -->
+## Continue reading
+
+Following tutorial will guide you through the configuration of node networking:
```
Comment (fedepaol) [inline docs/user-guide-101-reporting.md]: The following tutorial

**Comment by phoracek** [comment]
/override pull-kubernetes-nmstate-e2e-k8s

The issue seems to be unrelated. If I'm wrong, I will take the blame and fix it :(

**Pull request merged**